prime-router/src/test/kotlin/fhirengine/translation/RadxMarsTests.kt

```diff
@@ -0,0 +1,43 @@
+package gov.cdc.prime.router.fhirengine.translation
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNotEmpty
+import ca.uhn.hl7v2.DefaultHapiContext
+import ca.uhn.hl7v2.model.Message
+import ca.uhn.hl7v2.parser.CanonicalModelClassFactory
+import ca.uhn.hl7v2.util.Hl7InputStreamMessageIterator
+import ca.uhn.hl7v2.validation.impl.ValidationContextFactory
+import gov.cdc.prime.router.fhirengine.engine.encodePreserveEncodingChars
+import kotlin.test.Test
+import fhirengine.translation.hl7.structures.radxmars251.message.ORU_R01 as RADXMARS_ORU_R01
+
+class RadxMarsTests {
+    @Test
+    fun `test radxmars java class`() {
+        val rawMessage = """
+MSH|^~\&|MMTC.PROD^2.16.840.1.113883.3.8589.4.2.106.1^ISO|CAREEVOLUTION^00Z0000024^CLIA|AIMS.INTEGRATION.PRD^2.16.840.1.114222.4.3.15.1^ISO|AIMS.PLATFORM^2.16.840.1.114222.4.1.217446^ISO|20240404110305+0000||ORU^R01^ORU_R01|20240404110305_433848dc33ca4ce89ff4773fed48e8f6|P|2.5.1|||NE|NE|||||PHLabReport-NoAck^ELR251R1_Rcvr_Prof^2.16.840.1.113883.9.11^ISO
```

Comment:
Can we update one of the OBX segments to include the fields that we had to add?  Noticed this will setting up some testing